docs/source/conf.py

```diff
@@ -42,7 +43,9 @@
 
 # -- Options for HTML output -------------------------------------------------
 
-html_theme = "furo"
+import pydata_sphinx_theme
+
```

Comment:
```suggestion
```
you don't need this part

Reply:
I already deleted some parts needs force pushing